sw/device/lib/dif/dif_aes.c

```diff
@@ -466,3 +466,17 @@ dif_result_t dif_aes_read_iv(const dif_aes_t *aes, dif_aes_iv_t *iv) {
   }
   return kDifOk;
 }
+
+dif_result_t dif_aes_load_iv(const dif_aes_t *aes, const dif_aes_iv_t iv) {
+  if (aes == NULL) {
+    return kDifBadArg;
+  }
+
+  if (!aes_input_ready(aes)) {
+    return kDifUnavailable;
+  }
```

Comment:
That needs to be changed: the IV is only writable if AES is idle. So we need to check for the IDLE bit here, not the INPUT_READY bit. The input maybe writable while AES is busy...